### DISCUSSION TITLE
'type' object is not subscriptable

### DISCUSSION THREAD

**Discussion by AnsonHong703** [SirRender00/texasholdem#107] [General]
Hello SirRender00,
I found something wrong with running the texasholdem package. The screenshot is followed. I would appreciate it a lot if you could fix it.
![texasholdem_screenshot](https://user-images.githubusercontent.com/50323261/172042656-eeee4aad-981c-42f2-99f0-e3a1fe118e32.png)

**Comment by SirRender00** [comment]
Hey! Appreciate you commenting... I think what it is, is that type hinting using `list` instead of `List` was only introduced in python 3.9. While the `texasholdem` module purports to still support python 3.8..... apologies. The quickest fix would be to upgrade to python 3.9. 

Within the next week I'll add some tests for all the python versions, to see if they work as expected.

**Reply by SirRender00** [reply]
As a side... If we use `List` and `Dict` the python dependency can drop wayyy down. So I'll add your comment here as a `bug` and a feature to drop down to python (3.5? If I'm not mistaken).

**Comment by SirRender00** [comment]
Ok! Just merged in the changes! Should be good to go for python 3.8
